src/NuGet.Clients/PackageManagement.UI/PackageLoader.cs

```diff
@@ -485,7 +487,7 @@ private IEnumerable<string> GetSupportedFrameworks()
 
                 // if the package does not exist in the remote source, NuGet should
                 // try getting metadata from the local resource.
-                if (String.IsNullOrEmpty(metadata.Summary) && localResource != null)
+                if (String.IsNullOrEmpty(metadata?.Summary) && localResource != null)
```

Comment:
That's would also makes it super fast. What if local is empty? E.g. packages not restored